src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  }
+
+  if (days >= 3) {
+    return days * 40 - 20;
+  }
+
+  return days * 40;
```

Comment:
Consider using constants for the daily rental cost, long-term rental threshold and discount, and short-term rental threshold and discount. This will make your code more readable and maintainable, as per checklist item 2.